content/CORE/UIReference/TopMenu/UIPreferences.md

```diff
@@ -12,80 +16,82 @@ aliases: /core/administration/uipreferences/
 
 | Option | Style | Description |
 |--------|-------|-------------|
-| Choose Theme | drop down | Choose a preferred theme. Prebuilt and custom created themes are visible here. |
-| Prefer buttons with icons only | checkbox | Preserve screen space with icons and tooltips instead of text labels. |
-| Enable Password Toggle | checkbox | When set, an eye icon appears next to password fields. Clicking the icon reveals the password. |
-| Reset Table Columns to Default | checkbox | Reset all tables to display the system default columns. |
-| Retro Logo | checkbox | Revert branding back to FreeNAS. |
-| Reset All Preferences to Default | checkbox | Reset all user preferences to their default values. (Custom themes are preserved) |
-| UPDATE PREFERENCES | button | Apply the current checkbox settings to the web interface. |
+| **Choose Theme** | dropdown list | Choose a preferred theme. Prebuilt and custom created themes are visible here. |
+| **Prefer buttons with icons only** | checkbox | Preserve screen space with icons and tooltips instead of text labels. |
+| **Enable Password Toggle** | checkbox | Select to make an eye icon appear next to password fields. Click the icon to reveal the password. |
+| **Reset Table Columns to Default** | checkbox | Reset all tables to display the system default columns. |
+| **Retro Logo** | checkbox | Revert branding back to FreeNAS. |
+| **Reset All Preferences to Default** | checkbox | Reset all user preferences to their default values. (Custom themes are preserved) |
+| **UPDATE PREFERENCES** | button | Apply the current checkbox settings to the web interface. |
 
 ## Manage Custom Themes
 
 | Option | Style | Description |
 |--------|-------|-------------|
-| theme name (variable) | checkbox | Each custom theme is listed. |
-| DELETE SELECTED | button | Removes each selected custom theme from the system. |
-| CREATE NEW THEME | button | Opens the theme creation screen. |
+| **theme name (variable)** | checkbox | Each custom theme is listed. |
+| **DELETE SELECTED** | button | Removes each selected custom theme from the system. |
+| **CREATE NEW THEME** | button | Opens the theme creation screen. |
 
```

Comment:
Same comment here as for the previous table, remove the style column and incorporate whatever it is into the descriptions. For checkboxes descriptions can be something like this "Select to list each custom theme." if that is what the checkbox does. You might want to test what happens to verify what we said is true.
for buttons, "Click to remove each selected custom theme from the system." or is is true that you are deleting the selected style whether it is custom or not? Maybe test this to verify. Delete normally displays a confirmation box before it deletes what you selected as a failsafe.

Reply:
Manage Custom Themes: The new theme name displays with a selected checkbox next to it after a custom theme is created. Delete Selected button only activates after a custom theme is created. Clicking the Delete Selected button reverts the custom theme selection to default theme. Works as expected.